CONTRIBUTING.md

```diff
@@ -43,6 +43,38 @@ When creating an issue, please ensure you include the following information
 6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
 7. When all PRs are completed the issue is then closed.
 
+## Documentation impact
+
+## documenting your change
+
+When noting there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
+OR
+2. Contribute your own doc change to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc") and tag a relevant docs person for review and merge.
+
+## what to document
+
+Document changes if:
+
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+## how to document
+
+Docs work in:[docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
+
```

Comment:
Does this mean 'the documentation work is all done in [this](https://github.com/rackerlabs/docs-rpc) repo' ?